dist/aw-grid.css

```diff
@@ -550,8 +550,8 @@
         -webkit-box-flex: 0;
         -ms-flex: 0 0 auto;
             flex: 0 0 auto;
-        padding-left: .5rem;
-        padding-right: .5rem;
+        padding-left: -.5rem;
+        padding-right: -.5rem;
```

Comment:
The `padding-left` and `padding-right` values should be negative as described in [this PR](https://github.com/kristoferjoseph/flexboxgrid/commit/f36e4d364b87f0218226d66324d912e805a6ceca) from Flexbox Grid fixing the [Issue #75](https://github.com/kristoferjoseph/flexboxgrid/pull/75).